synced-cron/synced-cron-server.js

```diff
@@ -2,6 +2,7 @@
 SyncedCron = {
   _entries: {},
   running: false,
+  processId: `cron-${Date.now()}`,
```

Comment:
Is this an id? If yes, I would use Random.id()

Reply:
Done
https://github.com/quavedev/meteor-packages/commit/f78434daf327829a5defc0c2b9661a54a0cb54f3